openlibrary/templates/admin/sponsorship.html

```diff
@@ -0,0 +1,137 @@
+        <td>$summary['statuses'].get(status, 0)</td>
+      </tr>
+  </table>
+
+  <hr>
+
+  <table class="sponsor">
+    <tr>
+      <th>Internet Archive Revenue</th>
+      <td class="earnings--profit">\$$('{:,.2f}'.format(summary['total_cost_cents'] / 100.))</td>
+      <th>Better World Books Revenue</th>
+      <td class="earnings--profit">\$$('{:,.2f}'.format(summary['book_cost_cents'] / 100.))</td>
+    </tr>
+    <tr>
+      <th>Book Costs</th>
+      <td>\$$('{:,.2f}'.format(summary['book_cost_cents'] / 100.))</td>
+      <th>Est. Book Costs</th>
+      <td>\$$('{:,.2f}'.format(summary['est_book_cost_cents'] / 100.))</td>
+      <th>Δ Book Costs</th>
+      <td class="earnings--$('loss' if summary['delta_book_cost_cents'] < 0 else 'profit')">\$$('{:,.2f}'.format(summary['delta_book_cost_cents'] / 100.))</td>
+    </tr>
+    <tr>
+      <th>Scanning Costs</th>
+      <td>\$$('{:,.2f}'.format(summary['scan_cost_cents'] / 100.))</td>
+      <th>Est. Scanning Costs</th>
+      <td>\$$('{:,.2f}'.format(summary['est_scan_cost_cents'] / 100.))</td>
+      <th>Δ Scanning Costs</th>
+      <td class="earnings--$('loss' if summary['delta_scan_cost_cents'] < 0 else 'profit')">\$$('{:,.2f}'.format(summary['delta_scan_cost_cents'] / 100.))</td>
+    </tr>
+    <tr>
+      <th>Profit/Loss</th>
+      $ profit_loss = (summary['total_cost_cents'] - summary['book_cost_cents'] - summary['scan_cost_cents'])
+      <td class="earnings--$('loss' if profit_loss < 0 else 'profit')">\$$(profit_loss / 100.)</td>
+    </tr>
+  </table>
+
+  <hr>
+
+  <table class="sponsor">
+    <tr>
+      <th>Total Unscanned Books</th>
+      <td>$summary['total_unscanned_books']</td>
+      $if ctx.user and ctx.user.is_admin():
+        $ isbns = ','.join([s['identifier'].replace('isbn_', '') for s in summary['books']])
+        <td><a href="https://8882k.csb.app/#isbns=$isbns">Print Slips</a></td>
+    </tr>
+    <tr>
+      <th>Average Scan Cost</th>
+      $ num_total_books = len(summary['books'])
+      $ num_scanned_books = num_total_books - summary['total_unscanned_books']
+      $ avg_scan_cost = summary['scan_cost_cents'] / num_scanned_books
+      <td>\$$('{:,.2f}'.format(avg_scan_cost / 100.))</td>
+    </tr>
+    <tr>
+      <th>Est. Remaining Scanning Liability</th>
+      $ scan_cost_liability = avg_scan_cost * summary['total_unscanned_books']
+      <td class="earnings--$('loss' if scan_cost_liability > 0 else 'profit')">\$$('{:,.2f}'.format(scan_cost_liability / 100.))</td>
+    </tr>
+    <tr>
+      <th>Projected Profit/Loss</th>
+      $ adj_profit_loss = profit_loss - scan_cost_liability
+      <td class="earnings--$('loss' if adj_profit_loss < 0 else 'profit')">\$$(adj_profit_loss / 100.)</td>
+    </tr>
+  </table>
+  
+  <hr>
+
+  <table class="sponsor">
+    <thead>
+      <tr>
+        <th>#</th>
+        <th>Cover</th>
+        <th>IDs</th>      
+        <th>Title</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      $ i = 1
```

Comment:
Remove this `i`; webpy define `loop.index` inside the loop (1 indexed); use that instead.